example/awsv4_signing_example.py

```diff
@@ -0,0 +1,51 @@
+# example for google app engine
+
+import webapp2
+
+# needed to sign S3 policy
+import datetime
+import hashlib
+import hmac
+
+# get AWS details from somewhere...
+# from foo import YOUR_AWS_SECRET_KEY
+# from foo import YOUR_AWS_REGION
+
+
+class SignAuth(webapp2.RequestHandler):
+
+    def get(self):
+        to_sign = str(self.request.get('to_sign')).encode('utf-8')
+        
+        aws_secret = YOUR_AWS_SECRET_KEY
+        date_stamp = datetime.datetime.utcnow().strftime('%Y%m%d')
```

Comment:
As this is an example for evapoarte, we should use what evaporate sends. The below sample may need for decoration, but you probably get my gist.

```python
date_stamp = self.request.get('datetime')
```